src/vstart.sh

```diff
@@ -92,7 +92,11 @@ start_rgw=0
 ip=""
 nodaemon=0
 smallmds=0
-short=0
+if [ x`uname`x = xFreeBSDx ] ; then
```

Comment:
Fixed the xx's
And yes ZFS has long attributes, but Sage commented during Ceph day that we should use --short for some of the tests. And this is just to make sure that we do that when running FreeBSD, because is is known to sometimes fail the test in ceph-osd. This prevents that